app/assets/stylesheets/application.scss

```diff
@@ -6,3 +6,13 @@
 @import "neat";
 @import "base/base";
 @import "refills/flashes";
+
+a {
+  .b {
+    .c {
+      .d {
+        color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.